tests/test_pytorch/test_framework_dataset.py

```diff
@@ -2,20 +2,47 @@
 Test cases for framework specific ARMORY datasets.
 """
 
-import pytest
+import torch
 
 from armory.data import datasets
 from armory import paths
 
 DATASET_DIR = paths.DockerPaths().dataset_dir
 
 
-def test_pytorch_generator():
-    with pytest.raises(NotImplementedError):
-        _ = datasets.mnist(
-            split_type="train",
-            epochs=1,
-            batch_size=16,
-            dataset_dir=DATASET_DIR,
-            framework="pytorch",
-        )
+def test_pytorch_generator_cifar10():
+    batch_size = 16
+    dataset = datasets.cifar10(
+        split_type="train",
+        epochs=1,
```

Comment:
Need to verify how this works for multiple epochs as well

Reply:
It only generates a single epoch, regardless of what `epochs` is.

Reply:
You can call `iter(dataset)` again to start over, but the iteration is in the same order as the initial shuffle (it is not reshuffled).